shell.nix

```diff
@@ -16,14 +16,16 @@ pkgs.mkShell {
     minikube
     semver-tool
     yq-go
-  ];
+    which
+    curl
+    cacert
+    util-linux
+  ] ++ pkgs.lib.optional (builtins.getEnv "IN_NIX_SHELL" == "pure") docker;
   shellHook = ''
-    export HOME=${builtins.getEnv "HOME"}
     export GOPATH=$(pwd)/nix/.go
     export GOCACHE=$(pwd)/nix/.go/cache
     export TMPDIR=$(pwd)/nix/.tmp
     export PATH=$GOPATH/bin:$PATH
     mkdir -p "$TMPDIR"
-    make bootstrap
```

Comment:
What do you think @mhkarimi1383? Should we keep this automatically?

Reply:
shell.nix file made to make a development environment automatically for Nix users, So I think we should keep this to run automatically it will do nothing and just install requirements like other requirements that we are installing in the same file

Reply:
shell.nix file made to make a development environment automatically for Nix users, So I think we should keep this to run automatically it will do nothing and just install requirements like other requirements that we are installing in the same file

Reply:
A valid point, reverted.